hack/terraform/aws/variables.tf

```diff
@@ -9,6 +9,14 @@ variable aws_vm_type {
   default = "t2.micro"
 }
 
+variable aws_win_vm_type {
+  default = "t3.medium"
```

Comment:
can you reduce the instance type to t2.medium?

Reply:
Will experiment locally and update here.